src/main/java/com/craftingguide/exporter/extensions/craftingguide/ItemIconDumperScreen.java

```diff
@@ -57,7 +60,10 @@ public void dumpItems(ModModel mod, Collection<ItemModel> items, AsyncStep dumpI
         if (this.isExporting()) throw new IllegalStateException("already exporting!");
         this.setIsExporting(true);
 
-        if (items.isEmpty()) return;
+        if (items.isEmpty()) {
+        	this.setIsExporting(false);
+        	dumpItemsStep.done();
+        }
```

Comment:
I'm not sure why you would have needed to change this.  Can you elaborate?

Reply:
at one point there was an issue for if a mod added no items and i think i did this because of that but i just tried it and it works without it (also usually most people just close the pull request except one time when i just changed a couple lines in a lang file in avaritia addons so thank you for not closing the pull request) also some of the tab issues seem to be just github being weird.

Reply:
never mind this has to be kept in case a mod adds no items.

Reply:
Got it.  Please double-check your whitespace settings in your editor.  This looks like 8 spaces in GitHub.  Most likely, that means you're using tabs in your editor.